src/Resources/Subscriptions/index.ts

```diff
@@ -1,29 +1,27 @@
 import Client from '../../Client';
 import AbstractResource from '../AbstractResource';
 import Addons from './Addons';
+import PricingPlans from './PricingPlans';
+import { HttpClientConfig, HttpClientResponse } from '../../Interfaces/HttpClient';
 
 export default class Subscriptions extends AbstractResource {
     public addons: Addons;
+    public pricingPlans: PricingPlans;
 
     constructor(api: Client) {
         super(api);
         this.api = api;
         this.initialise();
```

Comment:
I believe both of these are called by the super's constructor, so shouldn't be necessary

```suggestion
```

Reply:
Will fix these retrospectively in another PR